test/test_video_reader.py

```diff
@@ -1258,8 +1246,7 @@ def test_audio_present_pts(self):
                     set_video_backend(backend)
                     _, audio, _ = io.read_video(
                         full_path, start_offset, end_offset, pts_unit='pts')
-                    self.assertGreaterEqual(audio.shape[0], 1)
-                    self.assertGreaterEqual(audio.shape[1], 1)
+                    assert all(audio.shape[:2])
```

Comment:
here as well it will probably be more obvious to include the `>= 1` part in the assertion